### PR TITLE
Salaries

### PR DESCRIPTION
**Pull request by kjgross**
Add Salaries to the player collection for nba players, insert into mongo, then surface to FE.

### CODE REVIEW

**Comment by kjgross** [comment]
@blakeparkinson when you get out of the casino, check dis guy out!

**Comment by blakeparkinson** [comment]
Correct me if I'm wrong but I dont think the method that fetches salaries will be called if we pull directly from mongo right?

Anyway, I'm starting to think we should probably store these (this and the colors) in mongo tho. Since they are static and we will be doing unneeded computation each time that we don't have to if the data is already in mongo...thoughts?

File: views/partials/player_card.hbs [hunk 1]
```diff
@@ -36,6 +37,7 @@
             <td id="fg-percent">\{{percentageCalc average.field_goals_att average.field_goals_made}}</td>
             <td id="ft-percent">\{{percentageCalc average.free_throws_att average.free_throws_made}}</td>
             <td id="threepoint-percent">\{{percentageCalc average.three_points_att average.three_points_made}}</td>
+            <td id="salary">\{{salary}}</td>
```
Comment (blakeparkinson) [inline views/partials/player_card.hbs]: i dont see this being passed to the FE anywhere?

**Comment by kjgross** [comment]
That's correct. I store salary in mongo. Only time it won't show up is for the 3 teams currently in mongo that aren't pulling the Api. Moving forward, it won't be an issue, though, bc once it's in mongo once via the Api, we'll never need to go look again. 

I think colors should be stored in mongo too. Probably in the team collection

File: views/partials/player_card.hbs (hunk 1 above)
Comment (kjgross) [inline views/partials/player_card.hbs]: It's part of the players object. fE gets it the same way it gets these free throw % and such 

Kelsey 
Edmodo Customer Success

**Comment by blakeparkinson** [comment]
oh I was forgetting that it was part of the players object

**Comment by blakeparkinson** [comment]
murging

**Pull request merged**